packages/core/src/services/documents/create.ts

```diff
@@ -47,10 +63,9 @@ export async function createNewDocument(
           content ??
           `
 ---
-provider: ${env.DEFAULT_PROVIDER_ID}
-model: gpt-4o-mini
----
-        `.trim(),
```

Comment:
don't remove this jumpline so that users can start writing in a new line right away